lms/validation/authentication/_bearer_token.py

```diff
@@ -61,6 +61,7 @@ class BearerTokenSchema(PyramidRequestSchema):
 
     user_id = marshmallow.fields.Str(required=True)
     oauth_consumer_key = marshmallow.fields.Str(required=True)
+    application_instance_id = marshmallow.fields.Int(required=False, allow_none=True)
```

Comment:
Having both `oauth_consumer_key` and `application_instance_id` alongside in this PR.